pages/05.faq/default.md

```diff
@@ -51,7 +51,7 @@ rclone mount [your mount name]: [local path to be mounted] \
 ===
 
 * Q. **Does Kavita collect any data on me?**
-* A. Kavita by default will collect stats on your install, this can be turned off after the initial launch. All data is anonymized and contains no information about your filenames or IP. The Kavita team actively use this data to help design the UX and plan enhancements. Thank you for opting in, it really helps in the design and planning effort. You can view the code at any time [here](https://github.com/Kareadita/KavitaStats) Here is a [record](https://github.com/Kareadita/KavitaStats/blob/main/KavitaStats/Entities/StatRecord.cs) from our stats database:
+* A. Kavita by default will collect stats on your installation. This can be turned off after the initial launch. All data is anonymized and contains no information about your filenames or IP. The Kavita team actively uses this data to help design the UX and plan enhancements. Thank you for opting in, it really helps in the design and planning effort. You can view the code at any time [here](https://github.com/Kareadita/KavitaStats). Here is a [record](https://github.com/Kareadita/KavitaStats/blob/main/KavitaStats/Entities/StatRecord.cs) from our stats database:
```

Comment:
This is the info I wanted to reference in your other stat area. This is most up to date.